src/features/Tiles/PersonTile/styled.js

```diff
@@ -0,0 +1,117 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.section`
+    padding: 56px 0 0;
+`;
+
+export const PersonDetailsTile = styled.article`
+    background: ${({ theme }) => theme.colors.tile.background};
+    box-shadow: 0px 4px 12px rgba(186, 199, 213, 0.5);
+    border-radius: 5px;
+    min-height: 544px;
+    padding: 40px;
+    margin-bottom: 64px;
+
+    @media(max-width: ${({ theme }) => theme.breakpoints.medium}px) {
+        padding: 32px;
+        margin-bottom: 32px;
+    }
+
+    @media(max-width: ${({ theme }) => theme.breakpoints.small}px) {
+        min-height: 328px;
+        padding: 24px;
+        margin-bottom: 20px;
+    }
+
+    @media(max-width: ${({ theme }) => theme.breakpoints.xsmall}px) {
+        padding: 16px;
+    }
+`;
+
+export const PersonPhoto = styled.img`
+    max-width: 312px;
+    max-height: 464px;
+    border-radius: 5px;
+    margin-right: 40px;
+    float: left;
+
+    @media(max-width: ${({ theme }) => theme.breakpoints.small}px) {
+        width: 188px;
+        height: 280px;
+        margin: 0 20px 0 0;
+    }
+
+    @media(max-width: ${({ theme }) => theme.breakpoints.xsmall}px) {
+        width: 114px;
+        height: 169px;
+        margin: 0 16px 0 0;
+    }
+`;
+
+export const Content = styled.div``;
```

Comment:
no styling for Content... maybe it s enough to have just div in code?

Reply:
Styles added in a subsequent revision